TOC-tidb-cloud.md

```diff
@@ -24,19 +24,19 @@
     - [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer)
   - [Upgrade a TiDB Cluster](/tidb-cloud/upgrade-tidb-cluster.md)
   - [Delete a TiDB Cluster](/tidb-cloud/delete-tidb-cluster.md)
-- Manage Data
+- Migrate Data
```

Comment:
These changes also affect the index file?

Reply:
Done.